.github/workflows/ci.yaml

```diff
@@ -11,6 +11,7 @@ env:
   CARGO_INCREMENTAL: 0
   CARGO_NET_RETRY: 10
   RUN_SLOW_TESTS: 1
+  RUST_BACKTRACE: short
```

Comment:
The short backtraces have been broken for years. Doesn't hurt to use this though.